UPGRADE-2.0.md

```diff
@@ -0,0 +1,16 @@
+UPGRADE FROM 1.x to 2.0
+=======================
+
+### General
+
+ * The `BaseEnum` class has been renamed. You should use `AbstractBaseEnum` instead.
+
+### Method isValidValue
+
+ * The only difference is that number of arguments are differents.
+
+   | Old name | New name
+   | -------- | ---
+   | `isValidValue($value)` | `isValidValue($value, $strict = true)`
+
+The `$strict` variable check the types of the needle in the values
```

Comment:
I think this is not a BC-break, which means it should not be in the `UPGRADE` file